internal/lsp/rego/rego.go

```diff
@@ -56,6 +56,8 @@ func LocationFromPosition(pos types.Position) *ast.Location {
 func AllBuiltinCalls(module *ast.Module) []BuiltInCall {
 	builtinCalls := make([]BuiltInCall, 0)
 
+	bis := GetBuiltins()
```

Comment:
Loading the builtins once also seemed like a good plan here since we're now loading a copy.